webrtc.html

```diff
@@ -1251,30 +1280,96 @@
                 application. In privacy-sensitive contexts, browsers can
                 consider mitigations such as generating SDP matching only a
                 common subset of the capabilities.</p>
-                <p>If the SDP generation process failed for any other reason,
-                the user agent MUST reject the returned promise with an
-                <code>DOMException</code> object of type
-                <code>OperationError</code> as its argument.</p>
+                <p>When the method is called, the User Agent MUST run the
+                following steps:</p>
+                <ol>
+                  <li>
+                    <p>Let <var>connection</var> be the
+                    <code><a>RTCPeerConnection</a></code> object on which the
+                    method was invoked.</p>
+                  </li>
+                  <li>
+                    <p>Return the result of <a href=
+                    "#enqueue-an-operation">enqueuing</a> the following
+                    steps:</p>
+                    <ol>
+                      <li>
+                        <p>Let <var>p</var> be a new promise.</p>
+                      </li>
+                      <li>
+                        <p>In parallel, start the process to generate an SDP
+                        offer, as described in <span data-jsep=
+                        "createoffer">[[!JSEP]]</span>.</p>
+                        <ol>
+                          <li>
+                            <p>If the process to generate an SDP offer failed
+                            for any reason, or if the identity provider was
+                            unable to produce an identity assertion, the User
+                            Agent MUST queue a task that runs the following
+                            steps:</p>
+                            <ol>
+                              <li>
+                                <p>If <var>connection</var>'s [[<a>isClosed</a>]]
+                                slot is <code>true</code>, then abort these
+                                steps.</p>
+                              </li>
+                              <li>
+                                <p>If the identity provider was unable to
+                                produce an identity assertion, reject
+                                <var>p</var> with a <code>DOMException</code>
+                                object whose <code>name</code> attribute has
+                                the value <code>NotReadableError</code>, and
+                                abort these steps.</p>
+                              </li>
+                              <li>
+                                <p>Reject <var>p</var> with a
+                                <code>DOMException</code> object whose
+                                <code>name</code> attribute has the value
+                                <code>OperationError</code> and, abort these
```

Comment:
Comma in wrong place, though I just noticed this isn't an if-statement, and there are no more steps, so can we simply remove ", and abort these steps?"